mantidproject/mantid
# Flake8 issues inelastic direct

Flake8 Warnings have been resolved in the scripts/Inelastic/Direct tab.
In addition, so cleanup and improvement has been made to naming and coding standards were appropriate.

**To test:**

* Review code changes
* Ensure `ISISDirectInelastic` tests pass
* Ensure all Flake8 warnings have been removed from changed files

Resolves part of #20508

**Release Notes** 
*Internal code change - does not need to be added to the Release notes

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
